invenio_records_resources/links.py

```diff
@@ -135,3 +119,113 @@ def links(self):
             _links["next"] = next_link
 
         return _links
+
+
+class Linker:
+    """Linker class.
+
+    Generates all the links for a given namespace.
+    """
+
+    def __init__(self, link_builders):
+        """Constructor.
+
+        :param link_builders: dict(string, list<LinkBuilder>)
+        """
+        self.link_builders = link_builders
+
+    def links(self, namespace, identity, **kwargs):
+        """Returns dict of links."""
+        output_links = {}
+        for link_builder in self.link_builders[namespace]:
+            link = link_builder.route_to_link(identity, **kwargs)
```

Comment:
minor: This is calling a function that it does not implement. Should it have a default behaviour? or is ok to let it fail? If the second, maybe is a good idea to put in the docstring something like "It also requires a link builder..."

Reply:
Not sure I understand this one... Each link_builder is expected to have a `route_to_link` method. If it doesn't this should just fail.